README.md

```diff
@@ -385,6 +385,19 @@ counterReducer(0, add(4)); // => 4
 counterReducer(0, increment()); // => 1
 ```
 
+A default handler can be attached, which is invoked for all actions not associated with a handler.
+```ts
+const add = createAction('todos/ADD', length => length)<number>();
+const reset = createAction('', () => 0)<number>();
+const counterReducer = createReducer(0)
+  .handleAction(add, (state, action) => state + action.payload)
+  .defaultHandler((state, action) => state = action.payload);
+
+const store = createStore(counterReducer);
+store.dispatch(add(3));
+store.dispatch(reset());
+```
+
```

Comment:
This example is useless... it doesn't convey the use case behind this API.

Please replace with recipe example "Clear store on user sign out" from my post: https://github.com/piotrwitek/typesafe-actions/issues/156#issuecomment-494748961

Reply:
done https://github.com/piotrwitek/typesafe-actions/pull/264/commits/5914037560cfdde370904c4b5774e97def5629de

Reply:
Thanks :)